library/general/test/report_test.rb

```diff
@@ -121,4 +123,58 @@
       expect(subject.GetMessages(0, 1, 0, 0)).to match(/Message/)
     end
   end
+
+  describe ".Settings" do
+    DATA_PATH =  File.join(File.dirname(__FILE__), 'data')
+    let(:ay_profile) { File.join(DATA_PATH, 'ay_profile.yml') }
```

Comment:
The `YAML.load_file` call is repeated quite a lot in this file, you can put it directly here instead of repeating it the tests:
```ruby
let(:ay_profile) { YAML.load_file(File.join(DATA_PATH, 'ay_profile.yml')) }
```
```ruby
expect(subject.Export()).to match(ay_profile)
```